runtime/commonMain/src/kotlinx/serialization/TypeTokenResolving.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2017-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization
+
+import kotlinx.serialization.internal.*
+import kotlin.reflect.*
+
+/**
+ * Reified version of [serializerByKType], provided for convenience.
+ * [KType] is obtained via [typeOf] function, consult its documentation
```

Comment:
`consult` -> `refer to`.
Also, as this method will be much more popular, it is better to make its documentation self-contained.